input/resources/Specimen.StructureDefinition.json

```diff
@@ -54,28 +54,6 @@
         ],
         "mustSupport": true
       },
-      {
-        "id": "Specimen.extension:diagnosis.value[x].coding",
```

Comment:
Why did you delete this?

Reply:
Because this information is already stored in the extension, but for some reson this was the only extension where the contents were listed seperately, which lead to an error about conflicting paths (value[x].coding vs valueCoding) when combined with the new FSH definition of the extension.